arch/arm/src/stm32h7/stm32h7x7xx_rcc.c

```diff
@@ -826,8 +830,6 @@ void stm32_stdclockconfig(void)
       regval |= STM32_PWR_CR3_LDOEN | STM32_PWR_CR3_LDOESCUEN;
       putreg32(regval, STM32_PWR_CR3);
 
-#if 0
```

Comment:
I recall some boards having issues starting correctly with this.  I need to check my boards.

Reply:
So I _think_ that the change directly below this was the reason it wasn't working. @davids5 fixed this in stm32h7x3xx_rcc.c after this was blocked out. 

https://github.com/apache/incubator-nuttx/pull/3730/files#diff-60aa753b9f6076921466b4eda3676f762ea547fd270a563d5ca10b0d39204ef5R844

But yes, it would be good if you can validate this on hardware that you think once had the issue.